--- common.pri.old      2014-08-14 15:56:14.219803957 +0200
+++ common.pri  2014-08-14 15:55:39.936478782 +0200
@@ -1,6 +1,6 @@
 exists(config.pri):infile(config.pri, SOLUTIONS_LIBRARY, yes): CONFIG += qtlockedfile-uselib
 TEMPLATE += fakelib
-QTLOCKEDFILE_LIBNAME = $$qtLibraryTarget(QtSolutions_LockedFile-head)
+QTLOCKEDFILE_LIBNAME = $$qtLibraryTarget(QtSolutions_LockedFile-2.4)
 TEMPLATE -= fakelib
 QTLOCKEDFILE_LIBDIR = $$PWD/lib
 unix:qtlockedfile-uselib:!qtlockedfile-buildlib:QMAKE_RPATHDIR += $$QTLOCKEDFILE_LIBDIR
